packages/theme/src/react/index.js

```diff
@@ -19,34 +19,46 @@ export function withTheme(BaseComponent) {
   const name = getDisplayName(BaseComponent)
 
   const Forwarded = React.forwardRef((props, ref) => {
-    const themeName = useTheme()
-    return <BaseComponent {...props} ref={ref} themeName={themeName} />
+    const { enableBleutrals, ...rest } = props
+    const themeName = useTheme({ enableBleutrals })
+    return <BaseComponent {...rest} ref={ref} themeName={themeName} />
   })
   Forwarded.BaseComponent = BaseComponent
   Forwarded.displayName = `withTheme(${name})`
+  Forwarded.propTypes = {
+    enableBleutrals: PropTypes.bool
+  }
+  Forwarded.defaultProps = {
+    enableBleutrals: false
+  }
 
   return hoistNonReactStatics(Forwarded, BaseComponent)
 }
 
-export function useTheme() {
+export function useTheme(config) {
   const themeName = React.useContext(ThemeContext)
-  return themeName
+  const bleutralsAdjustedThemeName =
+    config && config.enableBleutrals ? themeName + 'Bleutrals' : themeName
+  return bleutralsAdjustedThemeName
 }
 
 // NOTE: anything that uses <Theme /> from context only (no withTheme), will be broken with this new impl
 // TODO: find this set of components ^ and update
 export default function Theme(props) {
+  const value = props.enableBleutrals ? props.name + 'Bleutrals' : props.name
   return (
-    <ThemeContext.Provider value={props.name}>
+    <ThemeContext.Provider value={value}>
```

Comment:
What if this value wasn't primitive one but rather a object, one with a key **theme** `light` or `dark` and another key **bleutrals** `true` or `false`? Then all anyone would need to do to upgrade theme in addition to phase 1 components.  The **withTheme** function could pass through this new value in addition to `themeName` that could be used in refactoring button link and text during phase 1.

Reply:
Agreed, easier if value was already an objective.  But since it's already a primitive, changing to an obj would be a breaking change.  And since essentially all components rely on Theme, how would one upgrade Theme in a project without upgrading all the rest of the DS components (and that means *we* also have to upgrade all of them)?